app/src/main/java/de/westnordost/streetcomplete/map/components/SceneMapComponent.kt

```diff
@@ -62,10 +62,13 @@ class SceneMapComponent (
         getBaseSceneUpdates() + sceneUpdates.map { SceneUpdate(it.key, it.value) }
 
     private fun getBaseSceneUpdates(): List<SceneUpdate> {
+        val activeTileSource = vectorTileProvider.let {
+            if (sceneMapComponent?.isAerialView == true) it.rasterLayerSource else it.baseTileSource
+        }
```

Comment:
You misunderstood something there. When the aerial view is active, **both** the raster layer source and the vector tile source is used.

Reply:
Ah, you're right, this should always be using the base. That won't work if we ever add another source that needs an api key (thus, two needed), but for now it's fine; I'll revert this part.

Reply:
Removed this commit.